src/purifier/purifier.js

```diff
@@ -122,32 +231,16 @@ function standardPurifyConfig() {
   return /** @type {!DomPurifyConfig} */ (config);
 }
 
-/**
- * Gets a copy of the map of allowed tag names (standard DOMPurify config).
- * @return {!Object<string, boolean>}
- */
-export function getAllowedTags() {
-  const allowedTags = {};
-  // Use this hook to extract purifier's allowed tags.
-  DomPurify.addHook('uponSanitizeElement', function(node, data) {
-    Object.assign(allowedTags, data.allowedTags);
-  });
-  // Sanitize dummy markup so that the hook is invoked.
-  DomPurify.sanitize('<p></p>');
-  Object.keys(BLACKLISTED_TAGS).forEach(tag => {
-    allowedTags[tag] = false;
-  });
-  // Pops the last hook added.
-  DomPurify.removeHook('uponSanitizeElement');
-  return allowedTags;
-}
-
 /**
  * Adds AMP hooks to given DOMPurify object.
  * @param {!DomPurifyDef} purifier
  * @param {!Document} doc
+ * @param {!AttributeRewrite|undefined} attrRewrite
  */
-function addPurifyHooks(purifier, doc) {
+function addPurifyHooks(purifier, doc, attrRewrite) {
+  // Monotonically increasing counter used for keying nodes.
+  let KEY_COUNTER = 1;
```

Comment:
This should be an instance variable on the `Purify` class.

Reply:
Done, moved `addPurifyHooks` and `addPurifyHooksTripleMustache` to be methods instead.